### PR TITLE
Feature/grammar 20180703

### PR DESCRIPTION
**Pull request by TallTed**
fixed some English

### CODE REVIEW

File: binsrc/vsp/admin/admin_dav/bridge.vspx [hunk 1]
```diff
@@ -1,5 +1,4 @@
 <?xml version="1.0"?>
-<?xml version="1.0"?>
```
Comment (smalinin) [inline binsrc/vsp/admin/admin_dav/bridge.vspx:2]: Why it was deleted ?

**Review by smalinin — CHANGES_REQUESTED** [review]
Please review update for `binsrc/vsp/admin/admin_dav/bridge.vspx`

File: binsrc/vsp/admin/admin_dav/bridge.vspx (hunk 1 above)
Comment (TallTed) [reply binsrc/vsp/admin/admin_dav/bridge.vspx:2]: There were originally two such lines.  There should be one.  (Accidental duplication of #750 within #760 may be reducing that to zero.)

Comment (smalinin) [reply binsrc/vsp/admin/admin_dav/bridge.vspx:2]: Ok

**Review by smalinin — APPROVED** [review]

**Pull request merged**